From bde210eac24154d6913408a740474c326176beda Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Sat, 6 Nov 2021 16:07:50 -0700
Subject: [PATCH 24/45] portenta-x8-uboot: update u-boot additions to DTS

- drop cpu-idle params from CPU nodes
- drop all assigned-clocks info from usdhc nodes
- drop otg nodes
- add back mmc-hs400-1_8v

Signed-off-by: Michael Scott <mike@foundries.io>
---
 arch/arm/dts/portenta-x8-u-boot.dtsi | 39 ++++++++++++----------------
 1 file changed, 16 insertions(+), 23 deletions(-)

diff --git a/arch/arm/dts/portenta-x8-u-boot.dtsi b/arch/arm/dts/portenta-x8-u-boot.dtsi
index c3300877d8..90b27823e7 100644
--- a/arch/arm/dts/portenta-x8-u-boot.dtsi
+++ b/arch/arm/dts/portenta-x8-u-boot.dtsi
@@ -42,6 +42,22 @@
 	u-boot,dm-spl;
 };
 
+&A53_0 {
+	/delete-property/ cpu-idle-states;
+};
+
+&A53_1 {
+	/delete-property/ cpu-idle-states;
+};
+
+&A53_2 {
+	/delete-property/ cpu-idle-states;
+};
+
+&A53_3 {
+	/delete-property/ cpu-idle-states;
+};
+
 &clk {
 	u-boot,dm-spl;
 	u-boot,dm-pre-reloc;
@@ -68,39 +84,16 @@
 	u-boot,dm-spl;
 };
 
-&usdhc1 {
-	u-boot,dm-spl;
-	assigned-clocks = <&clk IMX8MM_CLK_USDHC1>;
-	assigned-clock-rates = <400000000>;
-	assigned-clock-parents = <&clk IMX8MM_SYS_PLL1_400M>;
-};
-
 &usdhc2 {
 	u-boot,dm-spl;
 	sd-uhs-sdr104;
 	sd-uhs-ddr50;
-	assigned-clocks = <&clk IMX8MM_CLK_USDHC2>;
-	assigned-clock-rates = <400000000>;
-	assigned-clock-parents = <&clk IMX8MM_SYS_PLL1_400M>;
 	fsl,signal-voltage-switch-extra-delay-ms = <8>;
 };
 
 &usdhc3 {
 	u-boot,dm-spl;
 	mmc-hs400-1_8v;
-	assigned-clocks = <&clk IMX8MM_CLK_USDHC3>;
-	assigned-clock-rates = <400000000>;
-	assigned-clock-parents = <&clk IMX8MM_SYS_PLL1_400M>;
-};
-
-&usbotg1 {
-	dr_mode = "otg";
-        status = "okay";
-};
-
-&usbotg2 {
-        dr_mode = "host";
-        status = "okay";
 };
 
 &i2c1 {
-- 
2.35.1

